supplementary_style_guide/glossary_terms_conventions/general_conventions/f.adoc

```diff
@@ -75,6 +75,17 @@ Use the hyphenated "fault-tolerant" when using it as an adjective.
 
 *See also*:
 
+//Add FedRAMP terminology Issue #395
+[[fedramp]]
+==== image:images/yes.png[yes] FedRAMP® (noun)
+*Description*: The Federal Risk and Authorization Management Program (FedRAMP®) is managed by the FedRAMP Program Management Office. Use the ® symbol at first usage in a document. For additional information, see the link:https://www.fedramp.gov/assets/resources/documents/FedRAMP_Branding_Guidance.pdf[FedRAMP Brand Guide].
```

Comment:
```suggestion
*Description*: The Federal Risk and Authorization Management Program (FedRAMP®) is managed by the FedRAMP Program Management Office. Use the registration symbol (®) at first usage in a document. For additional information, see the link:https://www.fedramp.gov/assets/resources/documents/FedRAMP_Branding_Guidance.pdf[FedRAMP Brand Guide].
```

Reply:
@sbmetz I'd say "on first usage" instead of "at first usage" but other than that, LGTM :)

Reply:
@sbmetz @bredamc - there is actually a PR [1] by @rolfedh that is standardizing to "on first use", so I'd suggest going with that

[1] https://github.com/redhat-documentation/supplementary-style-guide/pull/439